### ISSUE TITLE
[Suggestion] Forcefield "bubble" for colonies.

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#5310]
When thinking of the point I brought up about how structures like stations wouldn't have access to refillable oxygen from a planet's atmosphere, it got me thinking about things like asteroids, moons, and no-atmo planets with the exact same issue. Which got me thinking of a possible later-stage evolution of the force field and its use for oxygen preservation. Why not bubbles? A configurable forcefield up to a certain diameter. Place it on an asteroid, power it up, set up some oxygen generation, and bam. You can make structures without needing to have everything enclosed at all times. Even if it would be smart to do so in case of plot-based failure.

![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/cd10894f-b951-4cc4-8bcf-54a16380edad)
![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/216f461a-5997-40b3-942f-8ba94c24f8bd)
![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/9e843275-7309-4f04-9136-5c1bc3cc620f)

**Comment by tsunamayo** [comment]
Yes I though about this also, as a forcefield bubble but also maybe a structural one. Maybe one day... thx

**Comment by BigBadKangaroo** [comment]
This would be an amazing addition!
